slack.ts

```diff
@@ -0,0 +1,69 @@
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.SLACK_DOMAIN;
+
+interface Config {
+    channels: string[];
+}
+
+async function sendMessage(message : string, config : any) : Promise<void> {
+    for (var i = 0; i < config.channels.length; i++) {
+        const msg = {
+            token: tkn,
+            channel: config.channels[i],
+            as_user: false,
+            username: "HackGT",
+            text: message
+        };
+
+        const qs = querystring.stringify(msg);
+
+        const res = await fetch(`https://${domain}.slack.com/api/chat.postMessage?${qs}`);
+        const json = await res.json();
+        console.log(json);
+    }
+
+}
+
+async function setup(): Promise<void> {
```

Comment:
Unfortunately new stuff was merged that changed the way plugins are structured, but it's only a small change, take a look at:
https://github.com/HackGT/buzzer/blob/master/server/plugins/Twitter.ts